lib/store/src/csf/testing-utils/index.ts

```diff
@@ -0,0 +1,132 @@
+import {
+  isExportStory,
+  AnyFramework,
+  AnnotatedStoryFn,
+  StoryAnnotations,
+  ComponentAnnotations,
+  ProjectAnnotations,
+  Args,
+  StoryContext,
+  Parameters,
+} from '@storybook/csf';
+import { composeConfigs } from '@storybook/preview-web';
+
+import { prepareStory } from '../prepareStory';
+import { normalizeStory } from '../normalizeStory';
+import { HooksContext } from '../../hooks';
+import { normalizeComponentAnnotations } from '../normalizeComponentAnnotations';
+import { getValuesFromArgTypes, normalizeProjectAnnotations } from '..';
+import type { CSFExports, TestingStoryPlayFn } from './types';
+
+export * from './types';
+
+if (process.env.NODE_ENV === 'test') {
+  // eslint-disable-next-line global-require
+  const { default: addons, mockChannel } = require('@storybook/addons');
+  addons.setChannel(mockChannel());
```

Comment:
I don't think this is necessary any more, I think we changed it so that we don't need a channel.

https://github.com/storybookjs/storybook/blob/89830ad76384faeaeb0c19df3cb44232cdde261b/lib/addons/src/index.ts#L75-L77